content/src/main/content/jcr_root/etc/demandware/.content.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why do we need this?

Reply:
Here is some info: https://github.com/adobe/commerce-salesforce/wiki/Set-up-&-configuration-instructions#initial-template-provisioning